packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Field from 'terra-form-field';
+import DatePicker from './DatePicker';
+
+const propTypes = {
+  /**
+   * The DatePicker identifier. Links the htmlFor of the field to the select identifier.
+   */
+  datePickerId: PropTypes.string.isRequired,
+  /**
+   * Whether the date input should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Error message for when the input is invalid. This will only be displayed if isInvalid is true.
+   */
+  error: PropTypes.node,
+  /**
+   * Error Icon to display when the input is invalid.
+   */
+  errorIcon: PropTypes.element,
+  /**
+   * An array of ISO 8601 string representation of the dates to disable in the picker.
+   */
+  excludeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * A function that gets called for each date in the picker to evaluate which date should be disabled.
+   * A return value of true will be enabled and false will be disabled.
+   */
+  filterDate: PropTypes.func,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
+  /**
+   * Whether or not to hide the required indicator on the label.
+   */
+  hideRequired: PropTypes.bool,
+  /**
+   * Whether or not the field is an inline field.
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Whether or not the field is invalid.
+   */
+  isInvalid: PropTypes.bool,
+  /**
+   * Whether or not the label is visible. Use this props to hide a label while still creating it on the DOM for accessibility.
+   */
+  isLabelHidden: PropTypes.bool,
+  /**
+   * An array of ISO 8601 string representation of the dates to enable in the picker. All Other dates will be disabled.
+   */
+  includeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Custom input attributes to apply to the date input. Use the name prop to set the name for the input.
+   * Do not set the name in inputAttribute as it will be ignored.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  inputAttributes: PropTypes.object,
+  /**
+   * The label of the form control children.
+   */
+  label: PropTypes.node.isRequired,
+  /**
+   * Attributes to attach to the label.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  labelAttrs: PropTypes.object,
+  /**
+   * An ISO 8601 string representation of the maximum date that can be selected.
+   */
+  maxDate: PropTypes.string,
+  /**
+   * An ISO 8601 string representation of the minimum date that can be selected.
+   */
+  minDate: PropTypes.string,
+  /**
+   * Name of the date input. The name should be unique.
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * A callback function to execute when a valid date is selected or entered.
+   * The first parameter is the event. The second parameter is the changed date value.
+   */
```

Comment:
Please include the [onBlur](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/src/DatePicker.jsx#L52) and [onFocus](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/src/DatePicker.jsx#L71) props that were recently merged.

Reply:
Done: https://github.com/cerner/terra-framework/pull/642/commits/9cd02301e80b98aa1af8dd879f017f0a741e6d09